deployment/ansible/district-builder.yml

```diff
@@ -0,0 +1,14 @@
+- hosts: all
+  become: True
+
+  pre_tasks:
+    - name: Update APT cache
+      apt: update_cache=yes cache_valid_time=3600
+
+  roles:
+    - { role: "azavea.ntp" }
+    - { role: "district-builder.environment" }
+    - { role: "azavea.python-security" }
+    - { role: "district-builder.docker" }
+    - { role: "district-builder.aws-cli" }
```

Comment:
My assumption is that we're going to include AWS integration in the repo, but if that's not the case and we're just going to have containers and handle deployment of those containers separately, this can be removed. Currently the aws cli isn't being used for anything so it's a no-cost deletion.

Reply:
We can keep it out for now.